examples/faster_rcnn/train.py

```diff
@@ -104,6 +110,12 @@ def transform(in_data):
             trigger=plot_interval
         )
 
+    trainer.extend(
+        DetectionVOCEvaluator(
+            test_iter, model.faster_rcnn, use_07_metric=True),
+        trigger=(args.step_size, 'iteration'),
+        invoke_before_training=False)
+
```

Comment:
How about running evaluator twice (when lr changes and when the training finishes)?